packages/modules/manage-versions/src/components/InformationPanel/ChangesetInformationPanel.tsx

```diff
@@ -0,0 +1,102 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import "./ChangesetInformationPanel.scss";
+
+import { SvgClose } from "@itwin/itwinui-icons-react";
+import {
+  IconButton,
+  InformationPanel,
+  InformationPanelBody,
+  InformationPanelHeader,
+  Text,
+} from "@itwin/itwinui-react";
+import React, { FC } from "react";
```

Comment:
Dont use `FC` please.. Dont need to implicitly define children.. its confusing.

Reply:
removed FC